language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/inlayhint/InlayHintProvider.java

```diff
@@ -101,7 +102,7 @@ public static List<InlayHint> getInlayHint(InlayHintContext context) {
             for (int argumentIndex = 0; argumentIndex < argList.size(); argumentIndex++) {
                 //Find the inlay-hint location for the argument
                 NonTerminalNode argument = argList.get(argumentIndex);
-                if (argument.kind() != SyntaxKind.POSITIONAL_ARG) {
+                if (argument.kind() != SyntaxKind.POSITIONAL_ARG && argument.kind() != SyntaxKind.REST_ARG) {
```

Comment:
Let's check for named-args directly

Reply:
We have decided not to support named args

Reply:
I mean break when an argument is a named-arg.